lib/colorize.js

```diff
@@ -8,27 +8,37 @@ const Theme = {
   '-': color.red
 }
 
-const subcolorizeToCallback = function (key, diff, output, color, indent) {
+const subcolorizeToCallback = function (options, key, diff, output, color, indent) {
   let subvalue
   const prefix = key ? `${key}: ` : ''
   const subindent = indent + '  '
 
+  const outputElisions = (n) => {
+    if (n < (options.maxElisions ?? Infinity)) {
```

Comment:
this syntax is a breaking change; see #79.

Reply:
oops! thanks for the heads-up, and for the quick fix @ewoudenberg. I've been travelling recently so missed this comment; sorry about the breakage.